src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const minD = 20;
+  const maxD = 50;
```

Comment:
```suggestion
  const minDiscount = 20;
  const maxDiscount = 50;
```
Не сокращай названия переменных